openj9-11-rhel8.yaml

```diff
@@ -58,7 +62,7 @@ modules:
   - name: cct_module
     git:
       url: https://github.com/jboss-openshift/cct_module.git
-      ref: 0.42.0
+      ref: 0.41.1
```

Comment:
How does this change relate to the Maven/find issue?

Reply:
In particular you will lose this fix
086e590 (tag: 0.42.0) Remove the maven.config file

Reply:
HI Jonathon, the new version of maven will have to run the `maven.config` by default, for example,
in the new Apache Maven 3.6.2 (Red Hat 3.6.2-5),
```
. /etc/java/maven.conf
```
instead of the old Apache Maven 3.6.2 (Red Hat 3.6.2-4)
```
if [ -f /etc/java/maven.conf ] ; then
    . /etc/java/maven.conf
  fi
```
Therefore, simply removing the ` . /etc/java/maven.conf` as we did in branch `0.42.x` or tag `0.42.1` would still fail the startup of maven, complaining no config file exists. This following is the new maven startup script:
```
if [ -z "$MAVEN_SKIP_RC" ] ; then

  if [ -f /etc/mavenrc ] ; then
    . /etc/mavenrc
  fi

  . /etc/java/maven.conf

  if [ -f "$HOME/.mavenrc" ] ; then
    . "$HOME/.mavenrc"
  fi

fi
```

So we are now using another approach by adding
```diff
+ - name: "MAVEN_SKIP_RC"
+    value: true
```
to skip the default startup script instead of removing the config file. 

Because the 0.42.x branch / tag is not needed by using `MAVEN_SKIP_RC `, we are switching back to 0.41.1 tag as you used for openjdk to keep consistent with openjdk images.

Reply:
Makes a lot of sense, thanks!